cf-agent/cf-agent.c

```diff
@@ -893,13 +894,57 @@ static void FreeFixedStringArray(int size, char **array)
 
 /*******************************************************************/
 
+/**
+ * Gracefully terminate all children processes.
+ */
+static void CleanupProcessTree(void)
+{
+#ifndef _WIN32
+    ClearProcessTable(); /* Make sure old table is not cached */
+
+    Item *procs = NULL;
+    pid_t this, parent = getppid();
```

Comment:
I don't like this particular assignment, it's easy to miss the `,` and think it's similar to the other multu-assignment below that assigns the same value to all the variables.